File: snippets/java/app/src/main/java/TokenizePost.java
/* (C)2024 */
import com.cohere.api.Cohere;
import com.cohere.api.requests.TokenizeRequest;
import com.cohere.api.types.TokenizeResponse;

public class TokenizePost {
    public static void main(String[] args) {
        Cohere cohere = Cohere.builder().clientName("snippet").build();

        TokenizeResponse response =
                cohere.tokenize(
                        TokenizeRequest.builder().text("tokenize me").model("command").build());

        System.out.println(response);
    }
}

File: snippets/node/tokenize-post.js
import { CohereClient } from 'cohere-ai';

const cohere = new CohereClient({ token: process.env.COHERE_API_KEY! });

(async () => {
  const response = await cohere.tokenize({
    text: 'tokenize me! :D',
    model: 'command', // optional
  });
  
  console.log(response);
})();
